components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -99,10 +99,10 @@
           <chef-table>
             <chef-thead>
               <chef-tr>
-                <chef-th class="no-border-th">Name</chef-th>
-                <chef-th class="no-border-th">Admin</chef-th>
-                <chef-th class="no-border-th">Projects</chef-th>
-                <chef-th class="three-dot-column no-border-th"></chef-th>
+                <chef-th class="no-border">Name</chef-th>
```

Comment:
Author note: add a component global `no-border` class so the CSS is only written once.